apps/search-poc/src/config.ts

```diff
@@ -22,7 +24,7 @@ export function getCollectionEModelsQuery(resourceId: string) {
     prefix schema: <http://schema.org/>
     prefix nexus: <https://bluebrain.github.io/nexus/vocabulary/>
     prefix prov: <http://www.w3.org/ns/prov#>
-    SELECT ?name ?brainRegionLabel ?self ?project ?createdAt WHERE {
+    SELECT distinct ?name ?brainRegionLabel ?self ?project ?createdAt WHERE {
```

Comment:
Can You Capitalise this please? Makes it easier to read

Reply:
👍